### PR TITLE
Add delegation to balance

### LINKED ISSUE #1245: Integrate Delegation output in Balance
At least add a `delegation` field to `RequiredStorageDeposit`?

### PR DESCRIPTION
**Pull request by Alex6323**
## Links to any relevant issues

Closes #1245

### CODE REVIEW

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Also need to actually use it when computing balances

**Comment by thibault-martinez** [comment]
Any news on this?

**Review by Thoralf-M — DISMISSED** [review]

**Review by DaughterOfMars — DISMISSED** [review]

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
I'd update the bindings as well since the change is super small

**Review by Thoralf-M — DISMISSED** [review]

**Review by DaughterOfMars — DISMISSED** [review]

**Review by DaughterOfMars — DISMISSED** [review]

**Review by thibault-martinez — DISMISSED** [review]

File: bindings/nodejs/lib/types/wallet/wallet.ts [hunk 1]
```diff
@@ -58,10 +58,10 @@ export interface BaseCoinBalance {
 
 /** The required storage deposit per output type */
 export interface RequiredStorageDeposit {
-    /** The required amount for Alias outputs. */
-    account: u64;
     /** The required amount for Basic outputs. */
     basic: u64;
+    /** The required amount for Alias outputs. */
```
Comment (Thoralf-M) [inline bindings/nodejs/lib/types/wallet/wallet.ts]:
```suggestion
    /** The required amount for Account outputs. */
```

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**